x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -55,6 +56,8 @@ export class CspPlugin
 
     core.savedObjects.registerType(cspRuleAssetType);
```

Comment:
we should remove the previous `cspRuleAssetType` and its usages, in this or another PR

Reply:
It will delete in the following PR:)